src/main/java/io/lyuda/jcards/Dice.java

```diff
@@ -14,6 +14,24 @@ public class Dice {
      * The amount of sides on the dice.
      */
     private int sides;
+    /**
+     * A cryptographically secure random number generator used for rolling the dice.
+     * Innitialized in the constructor.
+     */
+    private SecureRandom secureRandom;
+    /**
+     * A variable that holds the last result of the roll() method.
+     */
+    private int lastRoll;
+    
+    /**
+     * Default constructor that Constructs a new {@code Dice} with six sides.
+     */
+    public Dice()
+    {
+        sides = 6;
```

Comment:
Here, *constructor chaining* should be applied i.e.

    public Dice() {
        this(6);
    }

This way, only a single constructor needs to be maintained.